controllers/suite_test.go

```diff
@@ -55,6 +58,7 @@ import (
 
 const (
 	DeploymentNamespace = "testns"
+	MachineNamespace    = "openshift-machine-api"
```

Comment:
NIT: I suggest either:
- use the same var from the production code [here](https://github.com/medik8s/node-healthcheck-operator/pull/177/files#diff-42b14dc2a70877933db4647434fcbe9b65a8859c7ec1911b0c9da7361c8e98b1R65) 
- make private (in case you prefer to avoid coupling between test/production code)

Reply:
decoupling is intended indeed, will make private in a follow up